File: app/src/main/java/com/otraempresa/android/example/RequestSiftScienceSessionAsyncTask.java
package com.otraempresa.android.example;

import android.content.Context;

import com.kushkipagos.android.Card;
import com.kushkipagos.android.KushkiException;
import com.kushkipagos.android.Transaction;

class RequestSiftScienceSessionAsyncTask extends AbstractRequestSiftScienceSessionAsyncTask {
    Context AppContext;

    RequestSiftScienceSessionAsyncTask(Context context) {

        super(context);
        this.AppContext=context;
    }

    @Override
    protected Transaction requestToken(Card card) throws KushkiException {
        return kushki.requestToken(card,10,AppContext,true);
    }
}

File: app/src/main/java/com/kushkipagos/kushkidemo/services/RequestSiftScienceSessionAsyncTask.kt
package com.kushkipagos.kushkidemo.services

import android.content.Context
import com.kushkipagos.exceptions.KushkiException
import com.kushkipagos.kushkidemo.contracts.AbstractRequestSiftScienceSessionAsyncTask
import com.kushkipagos.models.Card
import com.kushkipagos.models.Transaction

class RequestSiftScienceSessionAsyncTask: AbstractRequestSiftScienceSessionAsyncTask {
    val _context: Context
    constructor(context: Context): super(context){
        _context = context
    }

    @Throws(KushkiException::class)
    override fun requestToken(card: Card): Transaction {
        return _kushki.requestToken(card, 10.0, _context, true)
    }
}